packages/askar/src/wallet/AskarWallet.ts

```diff
@@ -699,7 +813,9 @@ export class AskarWallet implements Wallet {
       uri,
       profile: walletConfig.id,
       // FIXME: Default derivation method should be set somewhere in either agent config or some constants
-      keyMethod: keyDerivationMethodToStoreKeyMethod(walletConfig.keyDerivationMethod) ?? StoreKeyMethod.None,
+      keyMethod:
+        keyDerivationMethodToStoreKeyMethod(walletConfig.keyDerivationMethod) ??
```

Comment:
Should we set the default in this method? So we have consistent default behavior?